AzureMonitorAgent/ama_tst/modules/install/check_pkgs.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from error_codes import *
+from errors      import error_info
+from helpers     import get_package_version, find_ama_version
+
+
+
+def check_packages():
+    # check azuremonitoragent rpm/dpkg
+    ama_vers = find_ama_version()
+    if (ama_vers == None):
+        return ERR_AMA_INSTALL
+    if (len(ama_vers) > 1):
+        return ERR_MULTIPLE_AMA
```

Comment:
Are we sure that multiple AMA folders in `/var/lib/waagent` is necessarily an error/indicative of an issue? Might be good to just check what folders/files are there in `/var/lib/waagent` after an upgrade of AMA versions -- the older folder or .zip may still be there.

Reply:
Ashley mentioned it's possible that users have multiple AMA folders in /var/lib/waagent, so I think it's necessary to have this change